src/UI/SingleCommandApplication.php

```diff
@@ -37,10 +37,19 @@ class SingleCommandApplication extends Application
      *
      * @param Command $command The single (accessible) command for this application
      * @param string  $version The version of the application
+     *
+     * @throws \InvalidArgumentException
      */
     public function __construct(Command $command, string $version = 'UNKNOWN')
     {
-        parent::__construct($command->getName(), $version);
+        $commandName = $command->getName();
+        if (null === $commandName) {
+            throw new \InvalidArgumentException('command cannot be null');
+        }
+
+        $name =
```

Comment:
what? :)

Reply:
Interesting... Fixed... but how could he pass the build earlier?

Reply:
I asked myself the same question...